pom.xml

```diff
@@ -2809,7 +2809,7 @@
         <hive.version>${hive23.version}</hive.version>
         <hive.version.short>2.3.4</hive.version.short>
         <hive.parquet.group>org.apache.parquet</hive.parquet.group>
-        <hive.parquet.version>1.8.1</hive.parquet.version>
```

Comment:
do you know why we did it at the first place?

Reply:
This was suggested by me since Hive parquet version is different from ours.